tools/charm_helpers_sync/charm_helpers_sync.py

```diff
@@ -34,7 +34,7 @@ def parse_config(conf_file):
     if not os.path.isfile(conf_file):
         logging.error('Invalid config file: %s.' % conf_file)
         return False
-    return yaml.load(open(conf_file).read())
+    return yaml.load(open(conf_file).read(), Loader=yaml.FullLoader)
```

Comment:
Again, I suggest we use `safe_load()` as we don't actually need full loader.

Reply:
Noted, thanks for the suggestion.